pages/index.tsx

```diff
@@ -1,82 +1,51 @@
 import React from 'react';
 import Head from 'next/head';
 import Link from 'next/link';
-import styled from 'styled-components';
-
 import Container from '@styles/Container.styles';
 
-const Title = styled.h1`
-    font-size: 4rem;
-    text-align: center;
-    margin-bottom: 0;
-`;
-
-const Description = styled.h3`
-    font-size: 1.5rem;
-    text-align: center;
-    font-weight: normal;
-    margin: 1rem 0;
-`;
+// Graphql
+import { useQuery } from 'urql';
+import { NAVIGATION_MENU_QUERY } from '../src/graphql/queries';
 
-const Code = styled.span`
-    background: #f2f2f2;
-    border-radius: 5px;
-    padding: 0.75rem;
-    font-family: Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono,
-        Bitstream Vera Sans Mono, Courier New, monospace;
-`;
+export default function Home(): JSX.Element {
+    const [result] = useQuery({
+        query: NAVIGATION_MENU_QUERY,
+        variables: { url: '+urlMap:/docs/latest/table-of-contents' }
+    });
```

Comment:
We need to do this on the server, not in the client, did you check the docs I linked?: https://formidable.com/open-source/urql/docs/advanced/server-side-rendering/#using-getstaticprops-or-getserversideprops